SteamcLog/Classes/Helper/LogLevelPreset.swift

```diff
@@ -9,57 +9,55 @@
 import Foundation
 
 public enum LogLevelPreset: String, Codable {
-    /// Disk: verbose, system: verbose, remote: none, analytics: disabled
-    case firehose
-    /// Disk: none, system: debug, remote: none, analytics: disabled
-    case develop
-    /// Disk: verbose, system: none, remote: warn, analytics: enabled
-    case releaseAdvanced
-    /// Disk: none, system: none, remote: warn, analytics: enabled
+
```

Comment:
was there a good argument for removing the comments here? I think theres a good case for keeping them around, since they show up in the doc comments when you select one of the presets in Xcode.

Reply:
It just seemed like duplication of the settings below, and a place where we may end up with discrepancies if it's changed in one place and not the other. It didn't occur to me that this was removing the linked comments in the predictive text (? I can't recall the actual term) functionality. I'll add them back.

Reply:
Added comments back!